CHANGELOG.md

```diff
@@ -203,7 +203,7 @@
   * [CHANGE] - Running scripts on OpenShift 4.9 no longer generates a WARNING message.
 
 * **Monitoring**
-  * [FIX] - Updated SAS Viya logo on the Grafana Welcome screen so that it is easier to see on darker background.
+  * [FIX] - Updated SAS Viya platform logo on the Grafana Welcome screen so that it is easier to see on darker background.
```

Comment:
Is the logo the "SAS Viya platform" logo or the "SAS Viya logo"?  I suspect/assume it is the latter.  If so, the addition of "platform" would be inappropriate.